elastic/integrations
# Update windows integration package spec to v3

## Proposed commit message

This commit updates the windows integration to package spec `v3.2.1` and makes all necessary changes to the dashboards and fields definitons.

## Checklist


~~## Author's Checklist~~

## How to test this PR locally

1. Build the windows package
2. Start a stack using `elastic-package`
3. Create a new policy
4. Add the Windows integration
5. Deploy the Elastic-Agent to a Windows host
6. Check the following dashboards are working:
   - [Windows powershell] Overview
   - [Metrics Windows] Services
   - [Windows AppLocker] Audited and Blocked Applications 

## Related issues
- Closes https://github.com/elastic/integrations/issues/10274

~~## Screenshots~~
